### ISSUE TITLE
Incorrect `rand` alias generation for `libsecp256k1` crate

### ISSUE THREAD

**Issue by nikclayton-dfinity** [google/cargo-raze#270]
Add `libsecp256k1 = "0.3.5"` to the dependencies section of the `Cargo.toml` and run `cargo raze`.

With v0.6.1 this generates `libsecp256k1-0.3.5.BUILD.bazel` with the following entry (hopefully irrelevant parts removed):

```
...

rust_library(
    name = "secp256k1",
    crate_type = "lib",

    ...
    
    aliases = {
        "@raze__rand__0_7_3//:rand": "rand_test",
    },
)
```

Note the `aliases` entry. This causes `rand` to appear as `rand_test` when building, and you get the following error:

```
% bazel build //cargo:libsecp256k1
...
INFO: From Compiling Rust lib secp256k1 v0.3.5 (15 files):
error[E0432]: unresolved import `rand`
  --> external/raze__libsecp256k1__0_3_5/src/lib.rs:34:5
   |
34 | use rand::Rng;
   |     ^^^^ use of undeclared type or module `rand`

error: aborting due to previous error
```

Edit the generated build file to remove or comment out the incorrect alias and the crate can build.

**Comment by UebelAndre** [comment]
Hey, just from me (a non maintainer), this was not ignored. I'm trying to get some other changes through and then I can help dig into this

**Comment by GregBowyer** [comment]
I think I have honed in on this one and should have a fix shortly, the downside is it relates to #187 and will be a bit of a rework
